src/components/admin/article/ArticleController.jsx

```diff
@@ -206,6 +287,18 @@ class ArticleController extends FalcorController {
       return;
     }
 
+    // Copy the array of tags.
+    let processedTags = _.clone(this.state.tags);
+    if (_.uniqWith(processedTags, _.isEqual).length !== processedTags.length) {
```

Comment:
I really like the use of `_.isEqual`, much cleaner than manually writing out the keys etc :)